src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTuples.vb

```diff
@@ -6744,6 +6744,188 @@ End Module
         End Sub
 
         <Fact>
```

Comment:
><Fact> [](start = 8, length = 6)

AttributeTests_Tuples.cs has a ton of tests, including roundtripping, for all the nasty corner-cases here.